Homework_1_final/HashSet/HashSet.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace HashSet
+{
+    class HashSet<T> : IHashSet<T>
```

Comment:
nope, this is actually not a enumerable collection, there is no way for using it like the following:
``` c#
    var set = new HashSet<int>();
    set.Add(1);
    set.Add(2);
    set.Add(3);
    foreach (var item in set)
        Console.WriteLine(item);
```